lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -1343,4 +1343,32 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
                                                           data, dimValueList);
         return success();
       });
+
+  patterns.onOp("Tile", 13,
+                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+                  Torch::ValueTensorType resultType;
+                  Value operand;
+                  Value repeatDims;
+                  if (binder.tensorOperands(operand, repeatDims) ||
+                      binder.tensorResultType(resultType))
+                    return failure();
+                  Torch::PrimTolistOp repeatDimsToList = rewriter.create<Torch::PrimTolistOp>(
+                  binder.getLoc(),
+                  Torch::ListType::get(rewriter.getType<Torch::IntType>()), repeatDims);
+
+                  rewriter.replaceOpWithNewOp<Torch::AtenTileOp>(
+                      binder.op, resultType, operand, repeatDimsToList.getResult(0));
+                  return success();
+                });
+
+  patterns.onOp(
+      "Sign", 13, [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+      Torch::ValueTensorType resultType;
+      Value operand;
+      if (binder.tensorOperand(operand) || binder.tensorResultType(resultType))
+          return failure();
+
+      rewriter.replaceOpWithNewOp<Torch::AtenSignOp>(binder.op, resultType, operand);
```

Comment:
Have you used clang-format?

Reply:
Oh, I had the impression that it would be part of the CI.
used now, thanks for pointing this out 🤝